test/common.js

```diff
@@ -12,6 +12,61 @@ exports.tmpDirName = 'tmp';
 exports.PORT = +process.env.NODE_COMMON_PORT || 12346;
 exports.isWindows = process.platform === 'win32';
 
+var rimrafSync = function(p) {
```

Comment:
can you change these to simple named functions please, assigning to variables is not idiomatic for this file